src/controller/cpp/Node.cpp

```diff
@@ -105,6 +105,14 @@ const double *Node::getOrientation() const {
   return wb_supervisor_node_get_orientation(nodeRef);
 }
 
+const double *Node::getPose() const {
+  return wb_supervisor_node_get_pose(nodeRef, NULL);
+}
+
```

Comment:
Using a default argument would save us from this implementation:
```suggestion
```

Reply:
See above